nucypher/crypto/kits.py

```diff
@@ -55,11 +54,55 @@ def __init__(self,
                  ciphertext=None,
                  signature=NOT_SIGNED) -> None:
 
+        if type(capsule) == bytes:
+            raise Exception
```

Comment:
- I typically prefer `isinstance` since it's a little faster and considers inheritance, but perhaps neither of those considerations matter here.  If you're going to stick with `type` I suggest going right for `type(capsule) is bytes`.
- Any meaningful exception to raise here, or perhaps a message?  If not consider a TODO.

Reply:
I think it is a remainder from sanity checks I used for testing. I will remove it completely - we shouldn't be passing bytes around like that anyway.

Reply:
Removed the check and added annotations